documents/FAQ.md

```diff
@@ -83,13 +83,28 @@ Here is an example launch.json file to run the pubsub sample
     * Device certificate
         * Intermediate device certificate that is used to generate the key below
         * When using samples it can look like this: `--cert abcde12345-certificate.pem.crt`
-    * Key files 
+    * Key files
         * You should have generated/downloaded private and public keys that will be used to verify that communications are coming from you
         * When using samples you only need the private key and it will look like this: `--key abcde12345-private.pem.key`
 
+
+### I would like to build a browser application and got Error "Property does not exist on type 'typeof import("<path>/node_modules/aws-crt/dist/**native**/*")
+
+The aws-iot-device-sdk-v2 library is consistent of two part, [node(native)](https://aws.github.io/aws-iot-device-sdk-js-v2/node/index.html) and [browser](https://aws.github.io/aws-iot-device-sdk-js-v2/browser/index.html). The library will access the **native** API by default. You can configure the path in the `tsconfig.json` file to ensure that the app utilizes the **browser** API.
```

Comment:
Extremely minor nitpick: `consistent of two part, ...` -> `consists of two parts, ...`